tests/test_load_results.py

```diff
@@ -24,3 +28,10 @@ def test_load_results():
     known_revision = "bf3bf13ab40c3157080a7ab344c831b9ad18b5eb"
     assert known_model in results
     assert known_revision in results[known_model]
+
+
+@pytest.mark.parametrize("model", MODELS)
+def test_load_results_from_datasets(model):
```

Comment:
This test does not belong here (this is for testing integration with MTEB's load_results function. These should however be compatible objects.